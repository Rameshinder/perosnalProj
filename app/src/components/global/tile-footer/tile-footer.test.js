import React from 'react'
import { shallow, mount } from 'enzyme'

import TileFooter from './index'

jest.mock('@anz/button', () => 'Button')

describe('TileFooter', () => {
  const app = shallow(<TileFooter />)

  test('renders properly', () => {
    expect(app).toMatchSnapshot()
  })

  it('TileFooter should render without crashing', () => {
    shallow(<TileFooter />)
  })

  it('TileFooter should render with action bar', () => {
    shallow(<TileFooter />)
  })

  it('TileFooter should render with action bar', () => {
    mount(<TileFooter showPrimaryButton showSecondaryButton />)
  })
})
