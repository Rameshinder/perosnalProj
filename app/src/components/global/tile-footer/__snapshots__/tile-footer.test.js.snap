// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TileFooter renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <TileFooter
    primaryButtonDisabled={false}
    secondaryButtonDisabled={false}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": Array [
        null,
        null,
      ],
    },
    "ref": null,
    "rendered": Array [
      null,
      null,
    ],
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": Array [
          null,
          null,
        ],
      },
      "ref": null,
      "rendered": Array [
        null,
        null,
      ],
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
